docs/upgrade/_common/upgrade-image-opensource.rst

```diff
@@ -0,0 +1,43 @@
+There are two alternative upgrade procedures:
+
+* :ref:`Upgrading ScyllaDB and simultaneously updating 3rd party and OS packages <upgrade-image-recommended-procedure>`. It is recommended if you are running a ScyllaDB official image (EC2 AMI, GCP, and Azure images), which is based on Ubuntu 20.04.
+
+* :ref:`Upgrading ScyllaDB without updating any external packages <upgrade-image-upgrade-guide-regular-procedure>`.
+
+.. _upgrade-image-recommended-procedure:
+
+**To upgrade ScyllaDB and update 3rd party and OS packages (RECOMMENDED):**
+
+.. versionadded:: 5.0
+
+Choosing this upgrade procedure allows you to upgrade your ScyllaDB version and update the 3rd party and OS packages using one command. 
+
+#. Update the |SCYLLA_REPO|_ to |NEW_VERSION|.
+
+#. Load the new repo:
+
+    .. code:: sh 
+    
+       sudo apt-get update
+
+#. Run the following command to update the manifest file:
+
+    .. code:: sh 
+    
+       cat scylla-packages-<version>-<arch>.txt | sudo xargs -n1 apt-get -y
```

Comment:
missing the install command here it should be
`cat scylla-packages-<version>-<arch>.txt | sudo xargs -n1 apt-get install -y`

Also there is an alternative solution suggested by @soyacz  please refer to https://github.com/scylladb/scylladb/issues/11456